deploy/charts/trust-manager/values.yaml

```
@@ -101,6 +101,18 @@ app:
     # -- If false, disables the default seccomp profile, which might be required to run on certain platforms
     seccompProfileEnabled: true
 
+secretTargets:
+  # -- If set to true, enable writing trust bundles to Kubernetes Secrets as a target
+  enabled: false
+  # -- If set to true, grant read/write permission to all secrets. Use it with CAUTIOUS!
+  #    When set, ignores authorizedSecrets list setting.
+  authorizedSecretsAll: false
+  # -- A list of secret names which trust-manager will be permitted to read and write across all namespaces. 
+  #    These will be the only allowable Secrets that can be used as targets. 
+  #    If the list is empty it means read/write access to no secrets.
```

suggestion: trust-manager will still be able to read some secrets from the "trust namespace" so that they can be used as `sources`.

```suggestion
  #    If the list is empty, trust-manager will not be able to write to secrets and will only be able to read secrets in the trust namespace for use as sources.
```

---

done